draft-ietf-tls-external-psk-importer.md

```diff
@@ -87,6 +87,47 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 - External Identity: The identity of an EPSK.
 - Imported Identity: The identity of a PSK as sent on the wire.
 
+#Binder Key
+To prevent PSK Importers from being confused with standard OOB PSKs we change the label used in the computation of the PSK binder key.
+In TLS 1.3 the  PSK binder key computation is defined as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" | "res binder", "")
+                 |                     = binder_key
+
+We replace the string "ext binder" with "imp binder".
+This means the binder key is now computed as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" 
+                                        | "res binder" 
+                                        | "imp binder", "")
+                 |                     = binder_key
+
+This change should not affect any of the securtiy properties or security proofs previously established.
+The only change that is not a previously valid TLS 1.3 trace we mostly focus our attention on the change to the `binder_key` computation.
+The new label prevents the key being used in other contexts, including for encrypting or decrypting messages produced elsewhere.
+
+1. Because any keys produced using the new label are independent of keys produced using the old labels a PSK binder produced using an Importer will fail to verify when checked with the old label.
```

Comment:
This entire list seems unnecessary. Let's remove it.

Reply:
I'm going through the eight security guarantees in order and arguing why this change shouldn't affect them. I should probably say that at the top.

Reply:
What eight properties? (Either way, I'm not sure it's needed here. Maybe in an appendix if anywhere.)